datalake/src/main/java/com/sequenceiq/datalake/service/sdx/status/SdxStatusService.java

```diff
@@ -26,10 +26,15 @@
 import com.sequenceiq.datalake.entity.DatalakeStatusEnum;
 import com.sequenceiq.datalake.entity.SdxCluster;
 import com.sequenceiq.datalake.entity.SdxStatusEntity;
+import com.sequenceiq.datalake.flow.chain.DatalakeResizeFlowEventChainFactory;
 import com.sequenceiq.datalake.flow.statestore.DatalakeInMemoryStateStore;
 import com.sequenceiq.datalake.repository.SdxClusterRepository;
 import com.sequenceiq.datalake.repository.SdxStatusRepository;
 import com.sequenceiq.datalake.service.sdx.SdxNotificationService;
+import com.sequenceiq.flow.core.FlowLogService;
+import com.sequenceiq.flow.domain.FlowChainLog;
+import com.sequenceiq.flow.domain.FlowLog;
+import com.sequenceiq.flow.service.flowlog.FlowChainLogService;
 
 @Service
 public class SdxStatusService {
```

Comment:
As a general concept I think `SdxStatusService` has nothing to do with flow logs and flow chains and should not deal with it at all. Currently I don't see where it will be used and it seems like a workaround for some limitation rather than clean solution. I would remove this logic which makes decisions based on flow logs and put it inside the other class which needs it (currently I don't see which one needs it).